apps/antalmanac/src/components/RightPane/SectionTable/SectionTable.tsx

```diff
@@ -64,12 +66,52 @@ const styles = {
     },
 };
 
+const TableHeadColumns: { value: string; label: string }[] = [
+    { value: 'sectionCode', label: 'Code' },
+    { value: 'sectionDetails', label: 'Type' },
+    { value: 'instructors', label: 'Instructors' },
+    { value: 'dayAndTime', label: 'Times' },
+    { value: 'location', label: 'Places' },
+    { value: 'sectionEnrollment', label: 'Enrollment' },
+    { value: 'restrictions', label: 'Rstr' },
+    { value: 'status', label: 'Status' },
+];
+
 const SectionTable = (props: SectionTableProps) => {
     const { classes, courseDetails, term, colorAndDelete, highlightAdded, scheduleNames, analyticsCategory } = props;
     const courseId = courseDetails.deptCode.replaceAll(' ', '') + courseDetails.courseNumber;
     const encodedDept = encodeURIComponent(courseDetails.deptCode);
     const isMobileScreen = useMediaQuery(`(max-width: ${MOBILE_BREAKPOINT}`);
 
+    const [columns, setColumns] = useState([
```

Comment:
I think that having a consolidated mapping of the selected columns would be better.

Reply:
Hopefully resolved by me removing that nonsense activeTableColumn code and instead just filtering during render